src/ppl/src/IOPlacer.cpp

```diff
@@ -2647,12 +2647,21 @@ void IOPlacer::placePin(odb::dbBTerm* bterm,
 
   excludeInterval(interval);
 
-  logger_->info(PPL,
-                70,
-                "Pin {} placed at ({:.2f}um, {:.2f}um).",
-                bterm->getName(),
-                getBlock()->dbuToMicrons(pos.x()),
-                getBlock()->dbuToMicrons(pos.y()));
+  odb::Point original_pos(x, y);
+  if (pos != original_pos) {
+    debugPrint(logger_,
```

Comment:
I don't think we need to move this to a debugPrint now that it is inside the if statement.

Reply:
This is enormously much less verbose :+1: 

I would normally only look at pin positins in the GUI.

I can't offer much in the way of judgement on this residual logging and how useful it is, I wouldn't expect to ever see it.

The only time I can think of to give pins an exact position is when I am saving the pin placement to a .tcl file and reusing it(locking pin placement to speed up builds or possibly combining pin placement results). Let's say I have an array with heterogenous elements and routing by abutment. I would then need to use pin placements from one type of element on another type of element. If pins were moved at all, there would be something wrong with my flow because the pins were originally automatically placed.